app/common_functions/check_for_id.py

```diff
@@ -0,0 +1,15 @@
+from flask import make_response, abort
```

Comment:
Great work putting this functions into their own file so you can use them throughout your program.